src/libraries/System.Private.CoreLib/src/System/Half.cs

```diff
@@ -1015,7 +1015,7 @@ public static explicit operator double(Half value)
                 exp -= 1;
             }
 
-            return CreateDouble(sign, (ushort)(exp + 0x3F0), (ulong)sig << 42);
+            return Math.CreateDouble(sign, (ushort)(exp + 0x3F0), (ulong)sig << 42);
```

Comment:
nit: `Math` is basically "frozen" and not a place we want to find new methods, even internal ones

These would be "better" as internal `double.Create` and `float.Create` methods instead